AnalyticsAdmin/src/V1alpha/GoogleAdsLink.php

```diff
@@ -250,34 +250,6 @@ public function setAdsPersonalizationEnabledValue($var)
         $this->writeWrapperValue("ads_personalization_enabled", $var);
         return $this;}
 
-    /**
-     * Output only. Email address of the user that created the link.
-     * An empty string will be returned if the email address can't be retrieved.
-     *
-     * Generated from protobuf field <code>string email_address = 6 [(.google.api.field_behavior) = OUTPUT_ONLY];</code>
-     * @return string
-     */
-    public function getEmailAddress()
-    {
-        return $this->email_address;
-    }
```

Comment:
This breaks BC and is causing our tests to fail.

As this library is pre-1.0 and is in BETA, we can merge this and break BC if we can confirm this change is intentional (and update the tests).